BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/RecentTransactionsActivity.java

```diff
@@ -0,0 +1,136 @@
+package nl.tudelft.ewi.ds.bankchain.activities;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.util.Log;
+import android.view.View;
+import android.widget.ExpandableListView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import nl.tudelft.ewi.ds.bankchain.R;
+import nl.tudelft.ewi.ds.bankchain.Tools;
+import nl.tudelft.ewi.ds.bankchain.bank.Bank;
+import nl.tudelft.ewi.ds.bankchain.bank.BankFactory;
+import nl.tudelft.ewi.ds.bankchain.bank.Environment;
+import nl.tudelft.ewi.ds.bankchain.bank.Transaction;
+
+public class RecentTransactionsActivity extends AppCompatActivity {
+
+    ExpandableListAdapter listAdapter;
+    ExpandableListView expListView;
+    List<String> listDataHeader;
+    HashMap<String, List<String>> listDataChild;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_recent_transactions);
+
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        retrieveRecentTransactions();
+    }
+
+
+    public void retrieveRecentTransactions() {
+        Environment v = new Environment();
```

Comment:
Yes, we could create a singleton manager so we don't have to create a new Bank every time the activity is opened. That is however not in the scope of this task.